app/views/rooms/show.html.erb

```diff
@@ -98,36 +98,30 @@
                 </div>
             </div>
             <hr>
-            <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
-                <ol class="carousel-indicators">
-                    <% @images.drop(1).count.times do |image| %>
-                    <li data-target="#carouselExampleIndicators" data-slide-to="#{image}"></li>
-                    <% end %>        
-                </ol>
-                <div class="carousel-inner">
-                    <div class="carousel-item active">
-                        <% @images.first(1).each do |image| %>
-                            <%= attachment_image_tag image, :image, class: "carousel_image"  %>
-                        <% end %>
-                    </div> 
-                    <% @images.drop(1).each do |image| %>
-                        <div class="carousel-item">
-                            <%= attachment_image_tag image, :image, class: "carousel_image" %>
-                        </div>
-                    <% end %>
-                </div>
-                <a class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev">
-                    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
-                    <span class="sr-only">Previous</span>
-                </a>
-                <a class="carousel-control-next" href="#carouselExampleIndicators" role="button" data-slide="next">
-                    <span class="carousel-control-next-icon" aria-hidden="true"></span>
-                    <span class="sr-only">Next</span>
-                </a>
-            </div>
             <div class="row">
-                <div class="col-12">
-                    <%= image_tag(@room.images.first.image ,class:'w-100') %>
+                <div class="col-lg-12">
+                    <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
+                        <div class="carousel slide" data-ride="carousel" id="carousel-1">
+                            <div class="carousel-inner" role="listbox">
+                                <% @carousel.each do |image| %>
+                                    <div class="carousel-item <%= image.id == @carousel.first.id ? 'active' : '' %>">
+                                        <%= image_tag image.image, class: "mw-100 d-block gallery h-100" %>
```

Comment:
please remove  `h-100 `from the class in order to fix this issue
![image](https://user-images.githubusercontent.com/73863408/123956135-eac8e900-d9dc-11eb-9096-a4509237661f.png)